tools/deployment/sparkClusterInVnetTemplate.json

```diff
@@ -98,7 +98,7 @@
    "id": "[resourceId('Microsoft.Network/virtualNetworks', concat(parameters('clusterName'),'-vnet'))]",
    "subnet": "[concat(resourceId('Microsoft.Network/virtualNetworks', concat(parameters('clusterName'),'-vnet')), concat('/subnets/', parameters('virtualNetworkSubnetName')))]"
   },
-  "scriptActionUri": "https://mmlspark.azureedge.net/buildartifacts/0.5/install-mmlspark.sh"
+  "scriptActionUri": "https://mmlspark.azureedge.net/buildartifacts/0.6/install-mmlspark.sh"
```

Comment:
I can make scriptActionUri a parameter so that user have the option to override it at deployment time.